### PR TITLE
Gdr 2419

### PR DESCRIPTION
**Pull request by gladkia**
# Description
Improve pkgdown site
## What changed?
- updated news and added reference sections
- Related JIRA issue:  GDR-2419
- pkgdown site link: https://gdrplatform.github.io/gDRutils/

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,6 @@ Its main responsibilities are to provide:
 - getting/setting/validation functionality for identifiers for downstream processing and analysis
 - convenience functions for switching between long table and SummarizedExperiment data structures
 - core fitting functions
+
```
Comment (j-smola) [inline README.md:9]: Could you add free line between these points above - to better readability on site?
```
Its main responsibilities are to provide:

- getting/setting/validation functionality for identifiers for downstream processing and analysis

- convenience functions for switching between long table and SummarizedExperiment data structures

- core fitting functions
```

File: NEWS.md [hunk 2]
```diff
@@ -1,504 +1,509 @@
...
+## gDRutils 1.1.2 - 2023-12-01
+* fix bug with refining rowData
+* extend the list of headers
 
-## 1.1.0 (2023-10-24)
-- release Bioc 3.18
+## gDRutils 1.1.1 - 2023-11-22
+* sync master with devel branch
+* update schema to support NA in reference division time
+* added minor fix in code styling
```
Comment (j-smola) [inline NEWS.md:25]:
```suggestion
* add minor fix in code styling
```

File: NEWS.md [hunk 3]
```diff
@@ -1,504 +1,509 @@
...
+* added new function `gemoetric_mean`
```
Comment (j-smola) [inline NEWS.md:26]:
```suggestion
* add new function `gemoetric_mean`
```

File: NEWS.md [hunk 4]
```diff
@@ -1,504 +1,509 @@
...
 
-## 0.1.0.38 (2022-03-30)
-- fix hardcoded identifiers in validate SE
+## gDRutils 0.1.0.38 - 2022-03-30
+* fix hardcoded identifiers in validate SE
 
-## 0.1.0.37 (2022-03-28)
-- added a space between two-word cotreatment identifiers
+## gDRutils 0.1.0.37 - 2022-03-28
+* added a space between two-word cotreatment identifiers
```
Comment (j-smola) [inline NEWS.md:250]:
```suggestion
* add a space between two-word cotreatment identifiers
```

File: _pkgdown.yml [hunk 5]
```diff
@@ -1,4 +1,71 @@
-url: https://gdrplatform.github.io/gDRutils/
+url: https://gdrplatform.github.io/gDRstyle/
 template:
   bootstrap: 5
 
+reference:
+- title: Assay names
```
Comment (j-smola) [inline _pkgdown.yml:6]: Perhaps you could add a brief description of the characteristics of the function group?

**Review by j-smola — COMMENTED** [review]
You have proposed a division of functions based on the files in which the functions are stored. As the result there is a long list of groups (IMO too long to be helpful for users)
Maybe:
1. "Convert to data.table" and "Convert to BumpyMatrix" could be "Converter"?  Also with "Flatten"?
2. "Concatenate SummarizedExperiment(s)", "Merge SummarizedExperiment(s)", "Metadata of SummarizedExperiment(s)" and "Split SummarizedExperiment(s)" - just "SE operators"?
3. "Validate identifiers", "Identifiers" and  "Prettify" - jus "Identifiers helpers/operators"? Maybe also with "Headers"?
4. "Experiment validators" and "Experiment helpers" - just "Experiment helpers">
5. etc.

PS: Did you catch that we have two function with almost the same name `gen_synthetic_data` and `get_synthetic_data`?

File: _pkgdown.yml [hunk 6]
```diff
@@ -1,4 +1,71 @@
-url: https://gdrplatform.github.io/gDRutils/
```
[the remark this answers is not in the document]
Comment (gladkia) [reply _pkgdown.yml:1]: Yes.
Example: https://github.com/insightsengineering/rtables/blob/main/_pkgdown.yml#L1
Documentation: https://pkgdown.r-lib.org/articles/linking.html#across-packages

File: README.md (hunk 1 above)
Comment (gladkia) [reply README.md:12]: done.

**Comment by gladkia** [comment]
> 3\. Identifiers helpers/operators

Great idea. Done. Much cleaner!

**Comment by gladkia** [comment]
> You have proposed a division of functions based on the files in which the functions are stored. As the result there is a long list of groups (IMO too long to be helpful for users) Maybe:
> 
> 1. "Convert to data.table" and "Convert to BumpyMatrix" could be "Converter"?  Also with "Flatten"?
> 2. "Concatenate SummarizedExperiment(s)", "Merge SummarizedExperiment(s)", "Metadata of SummarizedExperiment(s)" and "Split SummarizedExperiment(s)" - just "SE operators"?
> 3. "Validate identifiers", "Identifiers" and  "Prettify" - jus "Identifiers helpers/operators"? Maybe also with "Headers"?
> 4. "Experiment validators" and "Experiment helpers" - just "Experiment helpers">
> 5. etc.
> 
> PS: Did you catch that we have two function with almost the same name `gen_synthetic_data` and `get_synthetic_data`?

(1:4) Great idea. Done. Much cleaner!

File: README.md (hunk 1 above)
Comment (gladkia) [reply README.md:12]: done

File: _pkgdown.yml (hunk 5 above)
Comment (gladkia) [reply _pkgdown.yml]: We agree that titles are now quite descriptive.

**Review by bczech — APPROVED** [review]

**Pull request merged**